sklearn/svm/tests/test_svm.py

```diff
@@ -949,3 +949,53 @@ def test_decision_function_shape_two_class():
             clf = OneVsRestClassifier(estimator(
                 decision_function_shape="ovr")).fit(X, y)
             assert_equal(len(clf.predict(X)), len(y))
+
+
+def test_ovr_decision_function():
+    train_base_points = np.array([[1, 2], [2, 1]])
+
+    # For all the quadrants (classes)
+    x = np.vstack((
```

Comment:
@raghavrv yes, @amueller suggested so, but keeping two points (these two are support vectors) per class is to ensure the direction of decision boundaries are 45 degree with the coordinates, so then we can say the decision value at (10, 10) should be greater than that at (5, 5).